rm-element.py

ssjaryan opened the pull request:
Added a solution for LeetCode's 'Remove Element' problem

umohnani8 commented:
Your commit message is not descriptive enough. Please take a look at the assignment for example and amend your commit by EOD Wednesday October 2nd if you want full credit for the assignment.

ssjaryan commented:
Added a solution for LeetCode's 'Remove Element' problem
- Implemented a Python solution to remove all instances of a specified value from an array.
- Used a while loop to iterate through the array and remove the specified value.
- Tested the solution with a sample input nums = [3, 2, 2, 3] and val = 3, resulting in getting Test Passed

umohnani8 commented:
@ssjaryan so you have added a comment to this PR describing what you did, but you need to add it to the commit otherwise it won't show up in the commit history. Right now your commit messages are just `rm-element.py` and `update rm-element.py`.

Additionally, please squash your commits so you only have one commit with your changes and commit message. You can google how to do that.
